locales/de.json

```diff
@@ -867,11 +855,11 @@
 		"text": {
 			"date": "Aktualität",
 			"popularity": "Beliebtheit",
-			"provider": "Anbieter",
+			"provider": "Anbieter:in",
```

Comment:
is this a real person or doesn't need the :in like provider

Reply:
Adapted.